apps/tsilhqotin-language-hub/src/Components/Pages/Songs/Songs.tsx

```diff
@@ -1,18 +1,126 @@
-import './Songs.module.css';
+import { Card } from '@mui/material';
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+import { DataGrid, GridColDef, GridRenderCellParams, GridRowsProp } from '@mui/x-data-grid';
+import { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import { getConfig } from '../../../config';
+
+export type SongData = {
```

Comment:
Can we call this `SongViewModel` instead?

Reply:
Renamed to SongViewModel.